modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {useRef} from 'react';
+import {useNavigate} from 'react-router-dom';
+
+import useFormModal from '../../hooks/useFormModal';
+import useMutate from '../../hooks/useMutate';
+import i18n from '../../i18n';
+import {TestrayTask, testrayTaskImpl} from '../../services/rest';
+import {Action} from '../../types';
+import {TaskStatuses} from '../../util/statuses';
+
+const useTestflowActions = () => {
+	const {updateItemFromList} = useMutate();
+	const navigate = useNavigate();
+	const {forceRefetch, modal} = useFormModal();
+
+	const actionsRef = useRef([
+		{
+			action: (task) =>
+				navigate(
+					`/project/${task.build?.project?.id}/routines/${task.build?.routine?.id}/build/${task.build?.id}`
+				),
+			icon: 'page',
+			name: i18n.translate('view-associete-build'),
+		},
+
+		{
```

Comment:
Remove unnecessary line

```suggestion
		},
		{
```